src/B/B.js

```diff
@@ -1,9 +1,10 @@
 // @flow
-import styled, { type ReactComponentFunctional } from 'styled-components';
+import * as React from 'react';
+import styled from 'styled-components';
 
-type Props = {};
+export type Props = {};
 
-const B: ReactComponentFunctional<Props> = styled.b`
+const B: React.ComponentType<Props> = styled.b`
```

Comment:
Can we use `ComponentType` from `'../utils/type.flow'`?

Reply:
Using custom `ComponentType` from `'../utils/type.flow'` when there is **defaultProps** or **propTypes** defined in Component is a workaround to work with react flow type,